doc/user/content/release-notes.md

```diff
@@ -48,6 +48,13 @@ Wrap your release notes at the 80 character mark.
 
 {{% version-header v0.9.5 %}}
 
+- Improve the performance of the log introspection, which should result in
+  lower latency spikes increased throughput while issuing many small queries,
+  and lower memory consumption. Additionally, with this change, the content
+  of the views are consistent at the introspection interval boundaries. (In the
+  past, some logs would reveal more details about ephemeral dataflows and
+  operators.)
```

Comment:
```suggestion
- Improve the performance of [built-in sources and views](/cli/#introspection-sources)
  in the [system catalog](/sql/system-catalog), which should result in lower
  latency spikes and increased throughput when issuing many small queries,
  and reduce overall memory consumption. Additionally, the content of the views
  is now consistent at the introspection interval boundaries. Prior to this
  release, some views would reveal more details about ephemeral dataflows and
  operators.
```

I mainly wanted to respect

> Link to at least one page where users can learn more about either the change or the area which the change was made.

from the release notes guide (a comment at the top of the file) but then ended up changing some wording. Please feel free to ignore.

Reply:
Thanks for the suggestions, accepted!